### PR TITLE
Book borrowing

### PR DESCRIPTION
**Pull request by anhnt796**

### CODE REVIEW

File: app/controllers/books_controller.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@ class BooksController < ApplicationController
   before_action :book_support, only: %i(new edit create)
 
   def index
-    @books = Book.page(params[:page]).per(Settings.BOOK_PER_PAGE)
+    @books = Book.page(params[:page]).per(Settings.BOOK_PER_PAGE).book_info
```
Comment (canhchimmcodon) [inline app/controllers/books_controller.rb]: paginates_per Settings.BOOK_PER_PAGE ---> books.rb

File: app/views/books/_book.html.erb [hunk 2]
```diff
@@ -1,9 +1,14 @@
 <li>
   <%= link_to book.title, book %>
-  <% if !current_user.nil? && current_user.librarian? %>
-    <%= link_to t(".edit"), edit_book_path(book) %>
-    <%= link_to t(".delete"), book, method: :delete,
-        data: {confirm: t(".confirm")} %>
-    <%= link_to t(".copies"), copies_path(book_id: book.id) %>
+  <% if !current_user.nil? %>
```
Comment (canhchimmcodon) [inline app/views/books/_book.html.erb]: current_user.present?

**Comment by canhchimmcodon** [comment]
OK

**Comment by anhnt796** [comment]
Ready

File: app/controllers/registered_copies_controller.rb [hunk 3]
```diff
@@ -0,0 +1,33 @@
+class RegisteredCopiesController < ApplicationController
+  before_action :find_book, only: %i(new)
+
+  def new
+    @registered_copy = RegisteredCopy.new(
```
Comment (baclv) [inline app/controllers/registered_copies_controller.rb]: bỏ `()` bạn nhé

File: app/models/user.rb [hunk 4]
```diff
@@ -78,4 +78,8 @@ def create_card_token
     self.card_activation_token = User.new_token
     self.card_activation_digest = User.digest(card_activation_token)
   end
+
+  def card_id
```
Comment (baclv) [inline app/models/user.rb]: dùng delegate bạn nhé

**Comment by baclv** [comment]
commented

**Comment by anhnt796** [comment]
Ready

**Comment by baclv** [comment]
merged

**Pull request merged**